pages/national_resources/se_resources.md

```diff
@@ -35,7 +51,7 @@ national_resources:
       Your_tasks: [dmp, data_publication, sensitive]
     url: https://nbis.se/services/guidance-on-data-management
   - name: Swedish Pathogens Portal
-    description: The Swedish Pathogens Portal provides information, guidelines, tools and services to support researchers to utilise Swedish and European infrastructures for data sharing.
+    description: The Swedish Pathogens Portal provides information, guidelines, tools and services to support researchers to utilise Swedish and European infrastructures for data sharing related to pathogens.
```

Comment:
This sounds a bit confusing to me, maybe we can rephrase to something along the lines of "The Swedish Pathogens Portal provides researchers with information, guidelines, tools, and services that facilitate the use of Swedish and European data-sharing infrastructures for pathogen research." or "...services that enable effective use of national and European infrastructures for sharing data related to pathogens".